frontend-maven-plugin/src/main/java/com/github/eirslett/maven/plugins/frontend/mojo/NpmMojo.java

```diff
@@ -84,6 +102,19 @@ protected boolean skipExecution() {
 
     @Override
     public synchronized void execute(FrontendPluginFactory factory) throws Exception {
+        String nodeVersion = getNodeVersion(workingDirectory, this.nodeVersion, this.nodeVersionFile, project.getArtifactId(), getFrontendMavenPluginVersion());
```

Comment:
We'll need to update metrics too so we can say x% of time using from version manager